src/main/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionController.java

```diff
@@ -145,8 +146,11 @@ public ObjectionController(PluggableResponseEntityFactory responseEntityFactory,
         );
 
         try {
-            String objectionId = objectionService.createObjection(requestId, companyNumber, ericUserId, ericUserDetails);
-            ObjectionResponseDTO response = new ObjectionResponseDTO(objectionId);
+            ObjectionResponseDTO response = objectionService.createObjection(requestId, companyNumber, ericUserId, ericUserDetails);
+            ObjectionStatus objectionStatus = response.getStatus();
+            if (objectionStatus.isIneligibleStatus()) {
```

Comment:
@mattch1 did you see my other comment about renaming this method to just 'isIneligible()'? Would read better here.

Reply:
Seen an updated it.

Reply:
Thanks.